pkg/kubectl/resource/builder.go

```diff
@@ -803,3 +803,36 @@ func HasNames(args []string) (bool, error) {
 	}
 	return hasCombinedTypes || len(args) > 1, nil
 }
+
+// HasMultipleTypes returns true if the provided args contain multiple resource kinds
+func HasMultipleTypes(args []string) bool {
+	args = normalizeMultipleResourcesArgs(args)
+	rKinds := sets.NewString()
+	for _, arg := range args {
+		if arg == "all" {
+			return true
+		}
+		rTuple, found, err := splitResourceTypeName(arg)
+		if err != nil {
+			continue
+		}
+
+		// if tuple not found, assume arg is of the form "type1,type2,...".
+		// Since SplitResourceArgument returns a unique list of kinds,
+		// return true here if len(uniqueList) > 1
+		if !found {
+			if strings.Contains(arg, ",") {
+				splitArgs := SplitResourceArgument(arg)
+				if len(splitArgs) > 1 {
```

Comment:
If `arg` contains "," then this is already > 1, isn't it? Is this check here because of something like `pods,pods`?